lib/r10k/module/local.rb

```diff
@@ -0,0 +1,21 @@
+require 'r10k/module'
+
+class R10K::Module::Local < R10K::Module::Base
+
+  R10K::Module.register(self)
+
+  def self.implement?(name, args)
+    args.is_a? Symbol and args == :local
```

Comment:
Yes, we use :local and Forge modules together in our company for two months in production environments.